immer/config.hpp

```diff
@@ -22,11 +22,12 @@
 #endif
 #endif
 
-#ifdef __has_feature
-#if !__has_feature(cxx_exceptions)
-#define IMMER_NO_EXCEPTIONS
-#endif
-#endif
+// The detection below is clang specific. Disable it so we always compile with exceptions support.
+// #ifdef __has_feature
+// #if !__has_feature(cxx_exceptions)
+// #define IMMER_NO_EXCEPTIONS
+// #endif
+// #endif
```

Comment:
(optional per slack discussion.)
```suggestion
#if 0  // The detection below is clang specific. Disable it so we always compile with exceptions support.
#ifdef __has_feature
#if !__has_feature(cxx_exceptions)
#define IMMER_NO_EXCEPTIONS
#endif
#endif
#endif  // 0
```